Source/ACE.Server/Managers/EmoteManager.cs

```diff
@@ -943,11 +945,54 @@ public void ExecuteEmote(BiotaPropertiesEmote emote, BiotaPropertiesEmoteAction
                     actionChain.AddDelaySeconds(emoteAction.Delay);
                     actionChain.AddAction(sourceObject, () =>
                     {
-                        var player = targetObject as Player;
                         player.Session.Network.EnqueueSend(new GameMessageHearDirectSpeech(sourceObject, emoteAction.Message, player, ChatMessageType.Tell));
                     });
                     break;
 
+                case EmoteType.TurnToTarget:
+                    var creature = sourceObject is Creature ? (Creature)sourceObject : null;
+                    actionChain.AddDelaySeconds(creature.Rotate(player));
+                    actionChain.AddAction(sourceObject, () =>
+                    {
+                        creature.Rotate(player);
```

Comment:
Is this double-rotate intended?

With 2 calls to rotate, the server will be sending the TurnTo MovementEvent to the client, waiting until its complete, and then sending another TurnTo packet